getting_hired/preparing_for_job_search/professional_networking.md

```diff
@@ -0,0 +1,88 @@
+### Introduction
+
+A robust professional network is an irreplaceable asset on your journey to becoming a web developer, and beyond. As social beings we are wired to connect with others, receive help, and give help. This is no different in the software ecosystem where countless communities take part in the mutually beneficial transactions of networking.
+
+One significant benefit of your network is that it might help you land your first job in software. This is not a guarantee, but it is the lived experience of many developers. As with all things, you get back what you put in, so the more developed your network is, the better your odds it will provide a benefit for you. Making genuine connections with people takes time, however, so we recommend you start investing time and effort sooner than later.
+
+### Lesson overview
+
+This lesson covers the following topics:
+
+*   What a professional network is
+*   How a professional network might benefit you
+*   Strategies and actions to consider when building and maintaining your professional network
+
+### What is a professional network?
+
+Your professional network is not the people you follow on social media, nor is it necessarily your friends. It is the people with whom you have made a genuine connection, whether in-person or online, because you have a mutual interest and can gain more value through being connected than you would otherwise have alone. These are the people you can ask for help, and in turn you can help out later.
+
+The network that will help you on your journey to becoming a software developer will mostly consist of people working in software development and adjacent roles, like recruitment or management. Take stock of your network in this field now. Chances are you might benefit from expanding it?
+
+### Why is a professional network useful?
+
+You can probably think of instances where you were stuck on a task and asked someone for help. This was you exercising your network. You can do the same with your software developer network to help you with many things like gather information, make decisions, and find a job.
+
+When it comes to information gathering, search engines and AI can be very helpful. But for anything with a bit of nuance, it’s often easier to ask a real human. For example, you might want to conduct an informational interview with a senior developer to learn about what they do and how they broke into the industry. You might be seeking feedback on your portfolio. Or, looking to get some career progression advice. Interacting with someone who has experience in what you seek can be far more beneficial because you can form a dialogue, ask follow-up questions, and get real-time feedback. 
```

Comment:
```suggestion
When it comes to information gathering, search engines and AI can be very helpful. But when more nuance is needed, it can be easier to ask a real human. For example:
-  You might want to conduct an informational interview with a senior developer to learn about what they do and how they broke into the industry.
-  You might be seeking feedback on your portfolio.
-  You may be looking to get some career progression advice.
Interacting with someone who has experience in what you seek can be beneficial because you can form a dialogue, ask follow-up questions, and get real-time feedback. 
```